### ISSUE TITLE
Problem with Connected Textures

### ISSUE THREAD

**Issue by nikita488** [xy-project/Project-XY#6]
When Multi-Tank is not activated everything is normal:
![2015-08-24_20 18 29](https://cloud.githubusercontent.com/assets/6237881/9446988/7d512a2e-4a9d-11e5-82bd-918bb74bc591.png)
When i activate it:
![2015-08-24_20 18 33](https://cloud.githubusercontent.com/assets/6237881/9446993/88bf9b98-4a9d-11e5-8c0d-a06304321128.png)

**Comment by xy-project** [comment]
Yep, those are the shadow blocks. Will find a solution when I have time :P
